cincinnati/src/lib.rs

```diff
@@ -1169,13 +1193,22 @@ mod tests {
 
     #[test]
     fn serialize_graph_with_conditional_edges() {
-        let graph = generate_graph(true);
+        let graph = generate_graph(true, false);
         assert_eq!(
             serde_json::to_string(&graph).unwrap(),
             r#"{"nodes":[{"version":"1.0.0","payload":"image/1.0.0","metadata":{}},{"version":"2.0.0","payload":"image/2.0.0","metadata":{}},{"version":"3.0.0","payload":"image/3.0.0","metadata":{}}],"edges":[[0,1],[1,2],[0,2]],"conditionalEdges":[]}"#
```

Comment:
`"conditionalEdges":[]` is not a very interesting set of conditional edges.  Can we use a PromQL example in this `generate_graph` case?

Reply:
addressed